### ISSUE TITLE
Feature Request: PixiJS v8 Support

### ISSUE THREAD

**Issue by lunarraid** [jnsmalm/pixi3d#209]
With the new massive rewrite of PixiJS 8, it seems a good amount of work will need to be done to support the new paradigm. Is this project still maintained, and if so, what would it take for v8 support? I might be interested in collaborating on this if there is interest.

**Comment by jnsmalm** [comment]
I have too much other stuff going on to do that I'm afraid. Also, I don't really see the need of PixiJS 8 I general, why would I go with that instead of v7? (At least in the areas I'm working with). I guess it's to support WebGPU, but not sure if there is anything else?

**Comment by lunarraid** [comment]
Mostly because v8 is a complete rewrite  and they likely won't do any further major work on v7, so it will eventually code rot. There are a lot of optimizations in the new code and API. WebGPU is interesting because it offers some more possibilities. I also work with @pixi/react and that's being moved with a complete rewrite from v7, so I'll eventually have to make the transition myself. I get being busy though. I'm still working through a bunch of upgrades myself, but I was curious if there was interest here.

**Comment by jnsmalm** [comment]
Yes, I will definitely use v8 later. But right now v8 doesn’t really offer any new features (as far as I know) compared to v7. I’ll stick with v7 for a while more for the production apps. WebGPU works in all major browsers nowadays?

**Comment by lunarraid** [comment]
I'm not entirely certain on the browser support front, but one thing I am looking into is React Native's WebGPU rendering layer, which would allow me to stop needing to use a WebView with RN once I get it working.

**Comment by guillaumebrunerie** [comment]
PIXI v8 is a lot faster than v7 (and not only with WebGPU, also with the WebGL renderer). I guess this is the main reason why one would want to use it over v7. Especially when doing 3D, which is presumably more resource intensive than 2D in general, a faster renderer seems like a good argument to upgrade.

**Comment by petem24** [comment]
Any update on V8 support?

**Comment by jnsmalm** [comment]
Same status as previously. I have so much other stuff going on that I don't have any time left for a project of this magnitude. From what I understand though, PixiJS guys is working on adding 3D support to PixiJS v8, not sure with what kind of capabilities though.

**Comment by wrench1997** [comment]
> Yes, I will definitely use v8 later. But right now v8 doesn’t really offer any new features (as far as I know) compared to v7. I’ll stick with v7 for a while more for the production apps. WebGPU works in all major browsers nowadays?


They said that if WebGPU doesn't support it, it will use the older V7 rendering mode. Since support issues aren't a concern during development, migrating to V8 is lossless at the underlying level.

**Comment by nevermemo** [comment]
For anyone interested I found out that PixiJS development team is working on a built in 3D support. Who knows when it will come out though.
https://pixijs.com/blog/8.16.0
